### ISSUE TITLE
Windows CLI `tailscale status` does not print IP when it is the only device on a tailnet

### ISSUE THREAD

**Issue by awsong** [tailscale/tailscale#6680]
### What is the issue?

Compiled windows CLI from latest source. When I first login, `tailscale status` is missing IP address.
<img width="464" alt="image" src="https://user-images.githubusercontent.com/787544/206728588-13f0b198-d4e1-4aab-8fe4-ea37de138492.png">

Linux client compiled from same source tree doesn't have this problem. 

### Steps to reproduce

1. Remove all machines from my account, including "hello.ts.net "
2. Remove C:\ProgramData\Tailscale folder
3. Start tailscaled
4. `tailscale up --unattended` and follow login process
5. `tailscale status`

### OS

Windows

### OS version

Windows 10

### Tailscale version

1.35.0-dev20221209   tailscale commit: 76389d8baf942b10a8f0f4201b7c4b0737a0172c-dirty

### Bug report

BUG-7d504c1a2c9c8dba087e6ae60fa5a6642f4f41d00df55f67ef54e9196a2db1b9-20221209144617Z-b967d48c7d15a690

**Comment by DentonGentry** [comment]
Does it eventually get an IP address? The timing may be different between Linux and Windows.

**Comment by awsong** [comment]
It actually got the IP address, just not showing from `tailscale status` command. 

`ipconfig` can see the correct IP was assigned to tailscale0 interface.

**Comment by DentonGentry** [comment]
Just to be clear: did you compare similar steps with the released version?

```
C:\Users\dgentry>tailscale status
100.123.241.80  win10                dgentry@       windows -
```

**Comment by awsong** [comment]
> Just to be clear: did you compare similar steps with the released version?
> 
> ```
> C:\Users\dgentry>tailscale status
> 100.123.241.80  win10                dgentry@       windows -
> ```

No I didn't, will do it now.

**Comment by awsong** [comment]
Tried latest release version 1.34. Turned out that it only happens when using CLI. 

<img width="933" alt="image" src="https://user-images.githubusercontent.com/787544/207621363-a7ba193b-9d0f-4a79-ac20-3309088a7055.png">

Did you remove all the machines including "hello.ts.net"? I didn't have this problem when I have at least one machine on my account.

**Comment by bradfitz** [comment]
I thought this would've been fixed by 0f604923d345fff3334d74e57492bfae98aba935 as part of fixing #4311 but apparently not.

I can still repro it at a270682e14e8eb11f4a098f28e5de96268818b83 today.

Super odd. No IPs anywhere:

```
$ tailscale ip
no current Tailscale IPs; state: Running

$ tailscale status --json
{
  "Version": "1.35.0-dev20230124-ta270682e1-dirty",
  "TUN": false,
  "BackendState": "Running",
  "AuthURL": "",
  "TailscaleIPs": null,
  "Self": {
    "ID": "nTVW1j1CNTRL",
    "PublicKey": "nodekey:37b15281279cdfa649add8391deaa6b5f6f5456c6d2465ff8ce440dfe4ed2979",
    "HostName": "tsdev",
    "DNSName": "tsdev.tail24754.ts.net.",
    "OS": "linux",
    "UserID": 68924249456576767,
    "TailscaleIPs": [],
    "Addrs": [
      "209.180.207.193:40961",
      "[2602:d1:b4cf:c100:f4fb:4ff:fe1a:4148]:35303",
      "10.0.0.52:40961",
      "172.17.0.1:40961",
      "[2602:d1:b4cf:c100:f4fb:4ff:fe1a:4148]:40961"
    ],
    "CurAddr": "",
    "Relay": "sea",
    "RxBytes": 0,
    "TxBytes": 0,
```

/cc @maisem @andrew-d @clairew
